smartmin/templates/smartmin/field.html

```diff
@@ -6,7 +6,7 @@
 {% if form_field and form_field.is_hidden %}
 {{ form_field }}
 {% else %}
-{% if form_field %}
+{% if form_field != None %}
```

Comment:
Looks like 1.9 has `is` but I guess we are not on that yet. Wonder where use of `!=` with None breaks, either way better than it was.